src/components/vm/disks/diskAdd.jsx

```diff
@@ -719,7 +747,9 @@ export class AddDiskModalBody extends React.Component {
                            <Button id={`${idPrefix}-dialog-add`}
                                    variant='primary'
                                    isLoading={this.state.addDiskInProgress}
-                                   isDisabled={this.state.addDiskInProgress || dialogLoading || (storagePools.length == 0 && this.state.mode != CUSTOM_PATH)}
+                                   isDisabled={this.state.addDiskInProgress || dialogLoading ||
+                                               (storagePools.length == 0 && this.state.mode != CUSTOM_PATH) ||
+                                               (this.state.mode == CUSTOM_PATH && this.state.device === "disk" && !this.state.customDiskVerified)}
```

Comment:
Should it? The text then would be "select a valid disk image or cdrom". Actually why did we decide on disabling the button again? Why can't we do the verification on submit hmmm?

Reply:
So, with the current code iteration we still disable the button, and in one scenario we can and up that the verification failed, if `head --bytes=16` throws an error. So I'll fix that to fallback to raw?

There is not much a user can do when it fails, and it should never happen.

Reply:
The main issue I have with this approach is that when you open a dialog and click on "Custom path", the user is immediately greeted with a disabled "Add" button, because they did not specify a file path yet.
![Screenshot from 2022-11-17 01-06-34](https://user-images.githubusercontent.com/42733240/202322107-24b4f49b-866a-48f3-972c-70e98b1ed187.png)

Whereas if they just choose "Create new", the "Add" button is not disabled, despite user not specifying volume name yet.
![Screenshot from 2022-11-17 01-41-16](https://user-images.githubusercontent.com/42733240/202326449-456de4d0-bb82-4f99-bef8-3cf20796d81d.png)


So this is kinda inconsistent.
And to me, this seems confusing from user's perspective as they do not know it's disabled because the "Custom path" input is empty, since there is no message there too. And they would not perhaps understand, why did "Add" button get disabled when they switched from "Create new" to "Custom path". Maybe they could even thing that the whole functionality is disabled for their VM?